README.md

```diff
@@ -207,82 +101,37 @@ Example:
 
 👤 **Author1**
 
```

Comment:
- [ ] Please since you are the only contributor to this project you can singularize the `Authors` section to `Author` and replace `Author1` with your name. :smile: this is not selected in this inline comments for this but please make sure you also remove the comment `Mention all of the collaborators of this project.` from your readme since it is showing in your readme and it is not necessary because it is a comment for you to get what should be done there.